tests/test_snapcraft_store_api.py

```diff
@@ -0,0 +1,138 @@
+from vcr_unittest import VCRTestCase
+from canonicalwebteam.store_api.stores.snapcraft import SnapcraftStoreApi
+
+
+class SnapcraftApiTest(VCRTestCase):
+    def setUp(self):
+        # configure self.attribute
+        self.client = SnapcraftStoreApi()
+        return super(SnapcraftApiTest, self).setUp()
+
+    def test_get_all_items(self):
+        result = self.client.get_all_items(size=16, api_version=1)
+        snaps = result["_embedded"]["clickindex:package"]
+        self.assertEqual(len(snaps), 16)
+
+        # Test different size
+        result = self.client.get_all_items(size=4, api_version=1)
+        snaps = result["_embedded"]["clickindex:package"]
+        self.assertEqual(len(snaps), 4)
+        [self.assertIn("snap_id", snap) for snap in snaps]
```

Comment:
If you're gonna do this test, I think you should do it both times - as in also on line 15 above

Reply:
You are right, done!